x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Logger } from '@kbn/core/server';
+import type { SecuritySolutionPluginRouter } from '../../../../types';
+import { registerSiemRuleMigrationsCreateRoute } from './create';
+
+export const registerSiemRuleMigrationsRoutes = (
+  router: SecuritySolutionPluginRouter,
+  logger: Logger
+) => {
+  registerSiemRuleMigrationsCreateRoute(router, logger);
+  // TODO: registerSiemRuleMigrationsGetRoute(router, logger);
```

Comment:
nit: Wdyt, maybe it's better to create a ticket and reference the future implementation there than keep a TODO in the comment?

Reply:
Yes, the ticket is already created, but it's in the security-team (private) repo. I'll remove the comment, it's a bit useless honestly, I know here is the place to register the routes 😅